Products/ZenEvents/zentrap.py

```diff
@@ -383,20 +412,32 @@ def oid2name(self, oid, exactMatch=True, strip=False):
 
         return oid
 
-    def _pre_parse(self, session, transport, transport_data, transport_data_length):
+    def _pre_parse(self, session, transport, transport_data,
+                   transport_data_length):
         """Called before the net-snmp library parses the PDU. In the case
         where a v3 trap comes in with unkwnown credentials, net-snmp silently
         discards the packet. This method gives zentrap a way to log that these
         packets were received to help with troubleshooting."""
         if self.log.isEnabledFor(logging.DEBUG):
-            ipv6_socket_address = c.cast(transport_data, c.POINTER(sockaddr_in6)).contents
+            ipv6_socket_address = c.cast(transport_data,
+                                         c.POINTER(sockaddr_in6)
+                                         ).contents
             if ipv6_socket_address.family == socket.AF_INET6:
-                self.log.debug("pre_parse: IPv6 %s" % (socket.inet_ntop(socket.AF_INET6, ipv6_socket_address.addr)))
+                self.log.debug("pre_parse: IPv6 %s",
+                               socket.inet_ntop(socket.AF_INET6,
+                                                ipv6_socket_address.addr)
+                               )
```

Comment:
This is an example of why I don't like the align with the opening '(' way of formatting.  I think the following is far more readable and is still pep8 compliant:
```
    self.log.debug(
        "pre_parse: IPV6 %s",
        socket.inet_ntop(socket.AF_INET6, ipv6_socket_address.addr)
    )
```
or, if it's still too wide:
```
    self.log.debug(
        "pre_parse: IPV6 %s",
        socket.inet_ntop(
            socket.AF_INET6, ipv6_socket_address.addr
        )
    )
```